std/traits.d

```diff
@@ -2527,8 +2527,10 @@ private enum NameOf(alias T) = T.stringof;
  * Get as an expression tuple the names of the fields of a struct, class, or
  * union. This consists of the fields that take up memory space, excluding the
  * hidden fields like the virtual function table pointer or a context pointer
- * for nested types. If $(D T) isn't a struct, class, or union returns an
- * expression tuple with an empty string.
+ * for nested types.
+ * Inherited fields (for classes) are not included.
+ * If $(D T) isn't a struct, class, or union, an
+ * expression tuple with an empty string is returned.
```

Comment:
This is a _really bad_ behavior for this case, but I guess there's nothing we can do about this anymore.

Reply:
Yep, more exhaustive testing would have uncovered that. Already the next item, RepresentationTypeTuple, is also not testing inherited classes, or static classes, ...